api/types/container/container_update.go

```diff
@@ -7,7 +7,7 @@ package container
 // See hack/generate-swagger-api.sh
 // ----------------------------------------------------------------------------
 
-// ContainerUpdateOKBody container update o k body
+// ContainerUpdateOKBody ContainerUpdateResponse
```

Comment:
These don't generate very nice Go docs. Why is the title just dumped here?

Reply:
Camel case type name just being split into words isn't a very good doc either. 
I don't know about the swagger go generator,   can have a look.

Reply:
This is produced by https://github.com/ndeloof/docker/blob/master/api/templates/server/operation.gotmpl#L23
`{{ template "docstring" . }}` which is itself implemented [here](https://github.com/go-swagger/go-swagger/blob/master/generator/templates/docstring.gotmpl)

My suggestion here is to add `description` in inline schemas for _actual_ documentation on types we want to see in generated code. WDYT ?

Reply:
Using just the description SGTM.